### ISSUE TITLE
Runtime error index out of range

### ISSUE THREAD

**Issue by roswellguitar** [peterbn/EDx52display#10]
Hello,

I tried launching the exe and got this error message when launching. I changed the config file to get the journal where it should find it. Do you have any idea where it could come from?

This is the result with --log debug

![image](https://user-images.githubusercontent.com/31155718/123641748-e1941c80-d822-11eb-80c9-70226b853553.png)

**Comment by RBracy** [comment]
Could you paste the contents of your config, also?

**Comment by roswellguitar** [comment]
journalsfolder: "E:\\Saved Games\\Frontier Developments\\Elite Dangerous"
refreshratems: 1000

**Comment by roswellguitar** [comment]
that's my conf.yaml if that's what you asked for?

**Comment by RBracy** [comment]
Let me get some clarity. Are you using single or double backslashes in your path name?  I attempted to insert some text to recommend you use two, and GitHub automatically made them single when my comment posted.

**Comment by roswellguitar** [comment]
i'm having double backslashes
i didn't even realized it deleted them on the post
![image](https://user-images.githubusercontent.com/31155718/123696886-8f6fed00-d85c-11eb-8c27-d2051f17f986.png)

**Comment by roswellguitar** [comment]
tried with only one backslashes and it's worst, idon't even have the command windows staying. it pops in and out immediately

**Comment by peterbn** [comment]
Hi - sorry for not responding earlier. 

You shouldn't really need to change the conf.yaml file. The default value should work. Have you tried with the default value?

```
journalsfolder: "%USERPROFILE%\\Saved Games\\Frontier Developments\\Elite Dangerous"
```

As for the command window not staying open, you can try to open a command line manually, navigating to where you have unzipped the tool, and then run the program.
That way, the command window will stay open even if the program crashes.

You can also try running it with `--log trace` to see the exact file it tries to read. That might give more information about what's going wrong.

**Comment by roswellguitar** [comment]
Hello Peter,

Problem is that my user is on drive E: and when using the default value
it's trying to get the

All tries :

*-FIRST TRY *
with config:

journalsfolder: "E:\\Saved Games\\Frontier Developments\\Elite Dangerous"
refreshratems: 1000

![image](https://user-images.githubusercontent.com/31155718/123769408-91bc6080-d8c9-11eb-9b7f-d6c5c5e7b331.png)

*-SECOND TRY*
with config:

journalsfolder: "E:\Saved Games\Frontier Developments\Elite Dangerous"
refreshratems: 1000

![image](https://user-images.githubusercontent.com/31155718/123769520-ac8ed500-d8c9-11eb-8641-821f229cab40.png)


*-THIRD TRY*

with default config

journalsfolder: "%USERPROFILE%\\Saved Games\\Frontier Developments\\Elite
Dangerous"
refreshratems: 1000

![image](https://user-images.githubusercontent.com/31155718/123769620-c6c8b300-d8c9-11eb-90ca-033eb1ef1451.png)


Best regards,
Maël

>

**Comment by peterbn** [comment]
Hello Maël,

Sadly, the images did not make it through to the bug tracker from your email. Would you mind logging in to github and adding them directly?

**Comment by roswellguitar** [comment]
Oups sorry, i updated my last post, tell me if it's not working

I'm wondering : does the journal logs have changed since odyssey update?

**Comment by peterbn** [comment]
not sure - I don't have Odyssey... Do you see files named `Journal.*.log` in either of the folders you tried? (* is a long sequence of numbers)

**Comment by roswellguitar** [comment]
Yes
![image](https://user-images.githubusercontent.com/31155718/123779418-70ac3d80-d8d2-11eb-9c0a-cb49d69f3f5e.png)

**Comment by peterbn** [comment]
Ok - Odyssey may be an issue, if it adds journal lines that I'm not considering. 

Would you mind compressing the entire folder as a zip and attaching it here? That _should_ let me try to reproduce later today.

**Comment by roswellguitar** [comment]
[Elite Dangerous.zip](https://github.com/peterbn/EDx52display/files/6732639/Elite.Dangerous.zip)

**Comment by roswellguitar** [comment]
Thanks for all your quick answers 👍 that's really kind of you!

**Comment by peterbn** [comment]
Not a problem - the zip file appears to be broken. I cannot open it with standard unzip tools :( 

Can you open it yourself?

**Comment by peterbn** [comment]
Hi - were you able to create a new zip file? The old one appears to be corrupted.

**Comment by Voodoo-le-Strange** [comment]
I encountered a very similar problem with Runtime Error: Index Out of Range

Here 's the screenshot I posted on the Elite Dangerous PVE forum, along with the zipped journals folder that Cmdr pakkenberg requested there.

[https://cdn.discordapp.com/attachments/624597132032344096/861179063628595220/Elite_X52_MFD.png](url)

**Comment by Voodoo-le-Strange** [comment]
I don't know if this is exactly the same problem, but I mentioned the issue about a Runtime Error: Slice Bounds Out of Range happening since the last Odyssey patch 5 was released. I mentioned in to Cmdr pakkenberg on the Elite: Dangerous PVE forum and he suggested I uploaded the journals.

So I have attached a screenshot of the error message, and uploaded the journals folder with the last couple of months files in it.

Hope this helps.

![image](https://user-images.githubusercontent.com/86926591/124394516-ba908b80-dcf7-11eb-9e88-ff0bc4a3f47a.png)

[Elite Dangerous.zip](https://github.com/peterbn/EDx52display/files/6760587/Elite.Dangerous.zip)

**Comment by roswellguitar** [comment]
Hello Peter,

Sorry for not ansering i was away this week-end. It seems i can open the .zip i gave in my last comment but i'm uploading it again.
Thanks Voodoo for your feeback! :)

[Elite Dangerous.zip](https://github.com/peterbn/EDx52display/files/6763084/Elite.Dangerous.zip)

**Comment by peterbn** [comment]
Thanks for the zip file Voodoo.

Maël, yours is still broken for some odd reason, but I'll see if I can't resolve the issue with what I got from Voodoo.

Completely unrelated, should I buy Odyssey? Is it worth playing?

**Comment by Voodoo-le-Strange** [comment]
This is a purely personal opinion, but Odyssey is fundamentally flawed compared to Horizons. Even after 5 major patches (which is pretty much all FDev have said they're doing until the console release in 6 months time) the lighting remains bad (too dark generally), the frame rate performance in stations is bad (so you have to brute force your frame rates with high end hardware or turn down quality to compensate), and they have a major problem with new planets surfaces being made up of custom tiling rather than procedurally generated as in Horizons (the issue with the tiling is that new planets have become quite bland and the tiling too often repeats noticeably). 

Personally, I think they have focused too much on getting the First Person Shooter aspect of the game, and the planetary atmospherics looking good at the expense of too many other things. Gaming websites are advising not to purchase the game until the console release...in the hope that Frontier have fixed all the flaws by then. I myself have gone back to Horizons, although most of my squadron are just braving it out. 

As I say, just my take on it, but the disappointment is echoed by many big-name influencers like Obsidian Ant and Down To Earth Astronomy, etc.

**Comment by peterbn** [comment]
Voodoo, check out https://github.com/peterbn/EDx52display/releases/tag/v1.3.2. Turns out your issue probably didn't have anything to do with Odyssey. It was "just" a bug in shortening the planet names by removing the system name that didn't correctly consider "Leesti" in the "Leesti" system.

Maël, you can try the release above as well to see if it solves your problem, but if it doesn't I do need a new zip file from you - or a .tar.gz or some other archive. I don't know what software you are using to create zip files, but could you try something else? I cannot open the second one you attached either. Let me know if you need suggestions for compression software.

**Comment by Voodoo-le-Strange** [comment]
I can confirm that the  "Slice Bounds Out of Range" issue is now fixed in v1.3.2. 
(Obviously just a complete coincidence me arriving in Leesti at the time the Odyssey patch dropped). 

Thank you very much for fixing, Peter. All the best.
